### PR TITLE
Added pygod as a package.

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sugatoray** [comment]
cc: @thewchan @jaimergp 

I opened this issue at the upstream (`pygod`).

- https://github.com/pygod-team/pygod/issues/28

**Comment by jaimergp** [comment]
I think it's ok to skip `pip check` here, either by commenting it out or adding a ` || true` construct.

**Comment by thewchan** [comment]
@jaimergp @sugatoray @conda-forge/staged-recipes I think this is ready for review; the window build fails due to a torch dependency, I can drop that one too if you all think appropriate

File: recipes/pygod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - scipy >=1.5.2
+    - setuptools >=50.3.1.post20201107
+
+test:
+  imports:
+    - pygod
+    - pygod.models
+    - pygod.utils
+  commands:
+    - pip list | grep -E "{{ name }}\s+{{ version }}"
```
Comment (ocefpaf) [inline recipes/pygod/meta.yaml:38]: You don't need this. Leaving the `pip check` commented out is fine. That will help us remember to re-enable it when upstream is fixed.

**Pull request merged**